packages/ui/src/hooks/use-sie.ts

```diff
@@ -19,7 +19,16 @@ export const useSieMethods = () => {
     socialConnectors: experienceSettings?.socialConnectors ?? [],
     signInMode: experienceSettings?.signInMode,
     forgotPassword: experienceSettings?.forgotPassword,
-    customContent: experienceSettings?.customContent,
+  };
+};
+
+export const useCustomSie = () => {
```

Comment:
how about we use one hook for all props as we can pick in the component? this should be good for now (as they are staying in the same context)

```ts
const useSignInExperience = () => {};
```

Reply:
Could, kind of forgot why I split them up.

Reply:
updated